components/expenses/ExpenseForm.js

```diff
@@ -798,6 +799,11 @@ const ExpenseFormBody = ({
                     <MessageBox type="info" fontSize="12px" mt="24px">
                       <FormattedMessage defaultMessage="Please make sure that all the expense items in this expense belong to the selected expense category. If needed, you may submit additional items in separate expenses with different expense categories." />
                     </MessageBox>
+                    {formik.values.accountingCategory?.instructions && (
+                      <MessageBox type="info" fontSize="12px" mt="24px">
+                        <HTMLContent content={formik.values.accountingCategory.instructions} />
+                      </MessageBox>
```

Comment:
Same here; we need a small adjustment.

![image](https://github.com/opencollective/opencollective-frontend/assets/1556356/eddc338b-e8de-4bdd-9ce3-b8df08abe699)